Makefile

```diff
@@ -52,7 +52,9 @@ mock_pkg:
 	mockery --filename=Azure.go --output=pkg/azure/mocks --dir=pkg/azure --outpkg=azure_mocks --name Azure
 
 mock_sql_cli:
-	mockery --filename="flow.go" --output="sql/mocks" --dir=sql/ --outpkg=mocks --name DockerBind
```

Comment:
You might wanna delete the mocks/flow.go file if its not longer going to be used

Reply:
@kushalmalani it has been renamed.

Reply:
@kushalmalani please merge the PR if it looks good to you. Thanks 🙏